analysis/blacklisted/blacklisted.go

```diff
@@ -252,11 +255,8 @@ func (b *Blacklisted) processURLs(urls chan UrlShort, waitgroup *sync.WaitGroup,
 		if b.safeBrowser != nil {
 			result, _ := b.safeBrowser.LookupURLs(urlList)
 			if len(result) > 0 && len(result[0]) > 0 {
-				for _ = range url.IPs {
-					score += 1
-					// TODO: Populate this with the information returned from the safebrowsing lookup.
-					sourcelist = append(sourcelist, "google-safebrowsing-api")
-				}
+				score = 1
```

Comment:
We need to look at the api results and adjust the score appropriately, but this is a quick fix since the IPs were duplicates.